code_generator_raw_uproot/servicex/templates/transform_single_file.py

```diff
@@ -26,20 +26,34 @@ def transform_single_file(file_path: str, output_path: Path, output_format: str)
 
         ttime = time.time()
 
-        if output_format == 'root-file':
+        if output_format in ('root-file', 'root-rntuple'):
```

Comment:
Why not put this as another top level output option? I get they are both `uproot`, but the actual code is quite different, and now this logic is buried.

Reply:
Discussed this privately.